src/remotestorage.js

```diff
@@ -763,8 +763,12 @@
     },
 
     localStorageAvailable: function () {
```

Comment:
`RemoteStorage.LocalStorage` is only for the caching of nodes and not for storing configuration data. It might not even be included in some builds (e.g. the `nocache` build).